README.md

```diff
@@ -16,17 +16,27 @@ Semref is an application that serves as a simulator of a microprocessor. It simu
 
 ## Technologies 
 
-* [Python3.6+](https://www.python.org/downloads/) - interpreted, high-level, general-purpose programming language 
+* [Python 2.7 - 3.5 - 3.6 - 3.7](https://www.python.org/downloads/release/python-374/) - interpreted, high-level, general-purpose programming language 
```

Comment:
Cannot use Python 2.7

Reply:
Thank you, just updated the link text to exclude Python version 2.7.